src/components/QuestionEdit/index.tsx

```diff
@@ -127,13 +137,25 @@ export default function QuestionEdit({
       <form
         aria-hidden={isPreview}
         className={classNames(styles.container, { [styles.toggledOff]: isPreview })}
-        onSubmit={handleSubmit}>
+        onSubmit={handleSubmit(onFormSubmit)}>
```

Comment:
Can't we execute onFormSubmit directly?

Reply:
no, we have to call it like this. See react-form-hook docs